src/types/mod.rs

```diff
@@ -28,3 +32,66 @@ pub struct VerifiedOutput {
     pub quote_body: QuoteBody,
     pub advisory_ids: Option<Vec<String>>,
 }
+
+impl VerifiedOutput {
+    pub fn to_vec(&self) -> Vec<u8> {
+        let mut bytes = Vec::new();
```

Comment:
nit: Can be changed to with_capacity

Reply:
I don't think we will able to know the exact length at compile time, because advisory IDs length can vary.